packages/lib-classifier/src/components/Classifier/components/TaskArea/TaskArea.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react'
+import { Tab, Tabs } from './components/Tabs'
+
+function TaskArea () {
```

Comment:
Would you consider bringing in the theme related styles I did for the redesign parts for the TaskArea already done in PFE? https://github.com/zooniverse/Panoptes-Front-End/blob/0ae10eed700f569fad857003f8bd43d96ab029b6/app/classifier/components/TaskArea/TaskArea.jsx#L6-L17

Of course it'll need a tiny update for the v2 grommet theme.